From 9e983c3f1772683a3d7a3da82bc326d5bcd6f7f1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 31 Dec 2024 15:02:15 +0100
Subject: [PATCH] theme-gtk: Use window icon for the menu button

This makes it consistent with the client-side decoration of GTK.
---
 libmetacity/meta-theme-gtk.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmetacity/meta-theme-gtk.c b/libmetacity/meta-theme-gtk.c
index ad687f89..e01e9c0c 100644
--- a/libmetacity/meta-theme-gtk.c
+++ b/libmetacity/meta-theme-gtk.c
@@ -847,7 +847,7 @@ meta_theme_gtk_draw_frame (MetaThemeImpl           *impl,
                 icon_name = "window-minimize-symbolic";
                 break;
               case META_BUTTON_TYPE_MENU:
-                icon_name = "open-menu-symbolic";
+                pixbuf = g_object_ref (mini_icon);
                 break;
               case META_BUTTON_TYPE_SPACER:
               case META_BUTTON_TYPE_LAST:
-- 
GitLab

